cli_tools/gce_vm_image_import/args/args.go

```diff
@@ -0,0 +1,289 @@
+//  Copyright 2020 Google Inc. All Rights Reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package args
+
+import (
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+	"time"
+
+	daisy_utils "github.com/GoogleCloudPlatform/compute-image-tools/cli_tools/common/utils/daisy"
+	"github.com/GoogleCloudPlatform/compute-image-tools/cli_tools/common/utils/param"
+	"github.com/GoogleCloudPlatform/compute-image-tools/cli_tools/gce_vm_image_import/importer"
+)
+
+// Flags that are validated.
+const (
+	imageFlag          = "image_name"
+	clientFlag         = "client_id"
+	dataDiskFlag       = "data_disk"
+	osFlag             = "os"
+	customWorkflowFlag = "custom_translate_workflow"
+)
+
+// ParsedArguments holds the structured results of parsing CLI arguments.
+type ParsedArguments struct {
+	Img         importer.ImageSpec
```

Comment:
I suggest to put this struct in package 'importer', and embed it as a whole in struct 'importer'.
Also, since img/env/translation are passed here and there, consider using pointer (e.g. img *ImageSpec) to ensure changed values are applied.

Reply:
`img/env/translation are passed here and there` -- this is specifically why it's not a pointer :)  I've been trying to be intentional about reserving pointers for methods and functions that mutate the underlying object. This is both for safety, as well as documenting intent.